example/rootfs/etc/services.d/example/finish

```diff
@@ -6,4 +6,4 @@
 if { s6-test ${1} -ne 0 }
```

Comment:
We should consider converting this one to Bash too

Reply:
Yea ok that makes sense, will adjust

Reply:
then we can also add nice bashio logs :D